### PR TITLE
Audit redesign font sizes and turn them into variables

### PR DESCRIPTION
**Pull request by thurab40**
This task is done

### CODE REVIEW

File: src/css/base/_body.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
 // Styleguide Base.body
 body {
 	background-color: $color-sand; // Extends background color to full width on an iPhone X in landscape mode: http://stephenradford.me/removing-the-white-bars-in-safari-on-iphone-x/
-	font-size: 1em;
+	font-size: $font-size-body;
```
Comment (ericwbailey) [inline src/css/base/_body.scss:9]: Can we revert this to `1em`? We're purposefully using an `em` unit on the body to better interface with any custom font sizing the user may have done.

File: src/css/components/_c-wcag-explanation.scss [hunk 2]
```diff
@@ -70,7 +70,7 @@
 
 .c-wcag-explanation__conformance-level {
 	display: block;
-	font-size: 1.15rem;
+	font-size: $font-size-body;
```
Comment (ericwbailey) [inline src/css/components/_c-wcag-explanation.scss:73]: Could you update this to use `$font-size-body-medium`?

File: src/css/screen.scss [hunk 3]
```diff
@@ -46,7 +46,8 @@
 	"logic/variables.font-tracking",
 	"logic/variables.font-weights",
 	"logic/variables.line-heights",
-	"logic/variables.global";
+	"logic/variables.global",
+	"logic/variables.font-sizes";
```
Comment (ericwbailey) [inline src/css/screen.scss:50]: Could you move this to above line 46 to put it in alphabetical order? So:

```
...
"logic/variables.font-families",
"logic/variables.font-sizes"
"logic/variables.font-tracking",
"logic/variables.font-weights",
...
```

Comment (thurab40) [reply src/css/screen.scss:50]: Cool, done. Thanks

**Comment by ericwbailey** [comment]
Amazing. Thank you so much!

**Pull request merged**